### PR TITLE
Feature/add user and owner endpoint

### PR DESCRIPTION
**Pull request by Asiasama710**

### CODE REVIEW

File: app/api/src/main/java/com/thechance/api/endpoints/Owner.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+fun Route.ownerRoutes(ownerUseCaseContainer: OwnerUseCaseContainer) {
+
+    route("/owner") {
+
+        post {
+            handleException(call){
+                val params = call.receiveParameters()
+                val name = params["name"]?.trim()
+                val password = params["password"]?.trim()
```
Comment (ahmednasserzaza) [inline app/api/src/main/java/com/thechance/api/endpoints/Owner.kt:20]: later we will ecyrypt password

File: app/core/src/main/java/com/thechance/core/data/datasource/OwnerDataSourceImp.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        return dbQuery {
+            val newOwner = OwnerTable.insert {
+                it[OwnerTable.ownerName] = ownerName
+                it[OwnerTable.password] = password
+                it[OwnerTable.isDeleted] = false
+            }
+            Owner(
+                ownerId = newOwner[OwnerTable.id].value,
+                userName = newOwner[OwnerTable.ownerName],
+                password = newOwner[OwnerTable.password],
```
Comment (ahmednasserzaza) [inline app/core/src/main/java/com/thechance/core/data/datasource/OwnerDataSourceImp.kt]: we shouldn't return password after create

File: app/core/src/main/java/com/thechance/core/data/usecase/user/UserUseCaseContainer.kt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package com.thechance.core.data.usecase.user
+
+import org.koin.core.component.KoinComponent
+
+data class UserUseCaseContainer(
```
Comment (ahmednasserzaza) [inline app/core/src/main/java/com/thechance/core/data/usecase/user/UserUseCaseContainer.kt:5]: Not neccessary to make container here you just has one useCase

**Review by ahmednasserzaza — COMMENTED** [review]
Well Done

File: app/core/src/main/java/com/thechance/core/data/usecase/user/UserUseCaseContainer.kt (hunk 3 above)
Comment (Asiasama710) [reply app/core/src/main/java/com/thechance/core/data/usecase/user/UserUseCaseContainer.kt:5]: But later won't we add delete User and  update?

**Review by NadaFeteiha — APPROVED** [review]

**Pull request merged**